faTraceback (most recent call last):
  File "/home/muhonja/PycharmProjects/backend-kl/main.py", line 1, in <module>
    from fastapi import FastAPI, Depends, HTTPException, File, UploadFile, Form
ModuleNotFoundError: No module named 'fastapi'
Traceback (most recent call last):
  File "/home/muhonja/PycharmProjects/backend-kl/main.py", line 1, in <module>
    from fastapi import FastAPI, Depends, HTTPException, File, UploadFile, Form
ModuleNotFoundError: No module named 'fastapi'
Traceback (most recent call last):
  File "/home/muhonja/PycharmProjects/backend-kl/main.py", line 1, in <module>
    from fastapi import FastAPI, Depends, HTTPException, File, UploadFile, Form
ModuleNotFoundError: No module named 'fastapi'
Traceback (most recent call last):
  File "/home/muhonja/PycharmProjects/backend-kl/main.py", line 1, in <module>
    from fastapi import FastAPI, Depends, HTTPException, File, UploadFile, Form
ModuleNotFoundError: No module named 'fastapi'
Traceback (most recent call last):
  File "/home/muhonja/PycharmProjects/backend-kl/main.py", line 1, in <module>
    from fastapi import FastAPI, Depends, HTTPException, File, UploadFile, Form
ModuleNotFoundError: No module named 'fastapi'
Traceback (most recent call last):
  File "/home/muhonja/PycharmProjects/backend-kl/main.py", line 1, in <module>
    from fastapi import FastAPI, Depends, HTTPException, File, UploadFile, Form
ModuleNotFoundError: No module named 'fastapi'
Traceback (most recent call last):
  File "/home/muhonja/PycharmProjects/backend-kl/main.py", line 1, in <module>
    from fastapi import FastAPI, Depends, HTTPException, File, UploadFile, Form
ModuleNotFoundError: No module named 'fastapi'
Traceback (most recent call last):
  File "/home/muhonja/PycharmProjects/backend-kl/main.py", line 1, in <module>
    from fastapi import FastAPI, Depends, HTTPException, File, UploadFile, Form
ModuleNotFoundError: No module named 'fastapi'
Traceback (most recent call last):
  File "/home/muhonja/PycharmProjects/backend-kl/main.py", line 1, in <module>
    from fastapi import FastAPI, Depends, HTTPException, File, UploadFile, Form
ModuleNotFoundError: No module named 'fastapi'
Traceback (most recent call last):
  File "/home/muhonja/PycharmProjects/backend-kl/main.py", line 1, in <module>
    from fastapi import FastAPI, Depends, HTTPException, File, UploadFile, Form
ModuleNotFoundError: No module named 'fastapi'
Traceback (most recent call last):
  File "/home/muhonja/PycharmProjects/backend-kl/main.py", line 1, in <module>
    from fastapi import FastAPI, Depends, HTTPException, File, UploadFile, Form
ModuleNotFoundError: No module named 'fastapi'
Traceback (most recent call last):
  File "/home/muhonja/PycharmProjects/backend-kl/main.py", line 1, in <module>
    from fastapi import FastAPI, Depends, HTTPException, File, UploadFile, Form
ModuleNotFoundError: No module named 'fastapi'
Traceback (most recent call last):
  File "/home/muhonja/PycharmProjects/backend-kl/main.py", line 1, in <module>
    from fastapi import FastAPI, Depends, HTTPException, File, UploadFile, Form
ModuleNotFoundError: No module named 'fastapi'
Traceback (most recent call last):
  File "/home/muhonja/PycharmProjects/backend-kl/main.py", line 1, in <module>
    from fastapi import FastAPI, Depends, HTTPException, File, UploadFile, Form
ModuleNotFoundError: No module named 'fastapi'
Traceback (most recent call last):
  File "/home/muhonja/PycharmProjects/backend-kl/main.py", line 1, in <module>
    from fastapi import FastAPI, Depends, HTTPException, File, UploadFile, Form
ModuleNotFoundError: No module named 'fastapi'
Traceback (most recent call last):
  File "/home/muhonja/PycharmProjects/backend-kl/main.py", line 1, in <module>
    from fastapi import FastAPI, Depends, HTTPException, File, UploadFile, Form
ModuleNotFoundError: No module named 'fastapi'
Traceback (most recent call last):
  File "/home/muhonja/PycharmProjects/backend-kl/test.py", line 3, in <module>
    jfjfhjhbfnnfkke
NameError: name 'jfjfhjhbfnnfkke' is not defined
Traceback (most recent call last):
  File "/home/muhonja/PycharmProjects/backend-kl/test.py", line 3, in <module>
    jfjfhjhbfnnfkke
NameError: name 'jfjfhjhbfnnfkke' is not defined
Traceback (most recent call last):
  File "/home/muhonja/PycharmProjects/backend-kl/test.py", line 3, in <module>
    jfjfhjhbfnnfkke
NameError: name 'jfjfhjhbfnnfkke' is not defined
